vms/evm/predicate/predicate_bytes.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package predicate
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/ava-labs/libevm/common"
+)
+
+// EndByte is used as a delimiter for the bytes packed into a precompile predicate.
+// Precompile predicates are encoded in the Access List of transactions in the access tuples
+// which means that its length must be a multiple of 32 (common.HashLength).
+// For messages with a length that does not comply to that, this delimiter is used to
+// append/remove padding.
+const EndByte = 0xff
```

Comment:
1. nit: if this is a delimiter why not name it `predicate.Delimiter`? If possible I want to see if we can avoid having multiple names for the same abstraction.
2. Why is this `0xFF`? It's arbitrary is my understanding?